src/components/chart/_macro-options.md

```diff
@@ -1,30 +1,32 @@
+| rangeAnnotations         | array               | false                  | An array of range annotations. Will be ignored if an iframe url is set.                                                                                                                                                                                                                                                                               |
+| referenceLineAnnotations | array               | false                  | An array of reference line annotations. Will be ignored if an iframe url is set.                                                                                                                                                                                                                                                                      |
+| estimateLineLabel        | string              | false                  | Label for the estimate line in the legend. Used in column charts with confidence intervals. Will be ignored if an iframe url is set.                                                                                                                                                                                                                  |
+| uncertaintyRangeLabel    | string              | false                  | Label for the confidence interval in the legend. Used in column charts showing uncertainty ranges. Will be ignored if an iframe url is set.                                                                                                                                                                                                           |
+| iframeUrl                | string              | false                  | A url for a visualisation to display in an iframe instead of displaying a Highcharts chart.                                                                                                                                                                                                                                                           |
+| footnotes                | `Object<Footnotes>` | false                  | Footnotes to appear below the chart in a 'details' element                                                                                                                                                                                                                                                                                            |
+| fallbackImageUrl         | string              | false                  | A url for a fallback image to display instead of the chart or iframed visualisation if JavaScript is disabled. If a fallback image is passed with an iframeUrl parameter, the iframe will be hidden for non-JavaScript uesrs. If it is not passed, it is assumed the iframed content will provide a fallback image to be displayed inside the iframe. |
```

Comment:
```suggestion
| fallbackImageUrl         | string              | false                  | A url for a fallback image to display instead of the chart or iframe visualisation if JavaScript is disabled. If a fallback image is passed with an iframeUrl parameter, the iframe will be hidden for non-JavaScript users. If it is not passed, it is assumed the iframe content will provide a fallback image to be displayed inside the iframe. |
```
Couple of typos

Reply:
Thanks. I was using 'iframed' in the sense of 'within the iframe' but happy to update.